openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -62,7 +62,7 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://media.sailthru.com/595/1k1/8/o/599f355101b3f.png" width="70"
+                                    src="https://edx-cdn.org/v3/prod/logo.png" width="70"
```

Comment:
Why is edX's URL hard-coded in the codebase? Can we pass the URL through as a parameter, like it's done for other URLs and variables in this file?

Reply:
Ah I'm not sure, I will review this and come back with an answer.